app/components/point-group.jsx

```diff
@@ -209,7 +211,7 @@ const PointGroup = props => {
               <div className={classes.selectPointsContainer}>
                 {groupedPoints.map((point, leadIndex) => {
                   return (
-                    <div key={point._id} className={classes.selectPoints}>
+                    <Level key={point._id} className={classes.selectPoints}>
```

Comment:
Can you try it with Level above groupedPoints.map? I think it would have the same effect, but fewer calls to Level. And the meaning would be "all these points are under the same level".

Reply:
And I mean for all the .map cases not just this one.